network-api/networkapi/buyersguide/utils.py

```diff
@@ -41,3 +52,38 @@ def database_backwards(self, app_label, schema_editor, from_state, to_state):
 
     def describe(self):
         return "Update %s bases to %s" % (self.name, self.bases)
+
+
+def convert_pni_file_to_product_page_image(pni_product: Product, new_product_page: ProductPage):
+    """
+    Take an existing product and convert it's FileField image to a WagtailImage object.
+
+    pni_product must be an instance of a Product
+    new_product_page must be an instance of a ProductPage
+    """
+    # 1. Get the mimetype of the image.
+    mime = MimeTypes()
+    mime_type = mime.guess_type(pni_product.image.file.name)  # -> ('image/jpeg', None)
+    mime_type = mime_type[0].split('/')[1].upper()
+
+    # 2. Create an image out of the FileField.
+    pil_image = PILImage.open(pni_product.image.file)
+    f = BytesIO()
+    pil_image.save(f, mime_type)
```

Comment:
if we're not using `f` anywhere ese anyway:
```suggestion
    pil_image.save(BytesIO() , mime_type)
```

Reply:
I was too hasty to resolve this. We do actually use `f` 

```python
    wagtail_image = WagtailImage.objects.create(
        title=new_image_name,
        file=ImageFile(f, name=new_image_name)
    )
``` 

I'm going to put this back in the next commit.